src/test/java/com/meetsky/step_definitions/Talk_StepDefinitions.java

```diff
@@ -1,79 +1,128 @@
 package com.meetsky.step_definitions;
 
+import com.meetsky.pages.BasePage;
+import com.meetsky.pages.LoginPage;
+import com.meetsky.pages.TalkPage;
+import com.meetsky.utilities.BrowserUtils;
+import com.meetsky.utilities.ConfigurationReader;
+import com.meetsky.utilities.Driver;
 import io.cucumber.java.en.And;
 import io.cucumber.java.en.Given;
 import io.cucumber.java.en.Then;
+import org.junit.Assert;
+
 
 public class Talk_StepDefinitions {
+
+    LoginPage loginPage = new LoginPage();
+    BasePage basePage = new BasePage();
+    TalkPage talkPage = new TalkPage();
+
+    @Given("User is logged in with valid credentials")
+    public void user_is_logged_in_with_valid_credentials() {
```

Comment:
Suggestion: You would have been used the LoginPage's login() method instead of this.